lnwire/channel_update.go

```diff
@@ -145,6 +149,7 @@ func (a *ChannelUpdate) Encode(w io.Writer, pver uint32) error {
 		a.HtlcMinimumMsat,
 		a.BaseFee,
 		a.FeeRate,
+		a.HtlcMaximumMsat,
```

Comment:
Similarly, should only be written if the flag is set.

Reply:
fixed